collector/src/compile/benchmark/codegen_backend.rs

```diff
@@ -0,0 +1,11 @@
+#[derive(Clone, Copy, Debug, Eq, Hash, PartialEq, clap::ValueEnum, serde::Deserialize)]
+#[value(rename_all = "PascalCase")]
+pub enum CodegenBackend {
+    LLVM,
```

Comment:
Standard Rust style would be to call this `Llvm`. Then you might not need the PascalCase?